Makefile

```diff
@@ -12,55 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Args:
-# GCE_PD_CSI_STAGING_IMAGE: Staging image repository
-REV=$(shell git describe --long --tags --match='v*' --dirty 2>/dev/null || git rev-list -n1 HEAD)
-ifdef GCE_PD_CSI_STAGING_VERSION
-	STAGINGVERSION=${GCE_PD_CSI_STAGING_VERSION}
-else
-	STAGINGVERSION=${REV}
-endif
-STAGINGIMAGE=${GCE_PD_CSI_STAGING_IMAGE}
-DRIVERBINARY=gce-pd-csi-driver
-DRIVERWINDOWSBINARY=${DRIVERBINARY}.exe
+.PHONY: push-multiarch-% push-multiarch
```

Comment:
With this change, there are other places need to be change for building and releasing image. The parameters passed will be different.